addons/crm/static/src/views/fields/statusbartime/statusbartime_field.js

```diff
@@ -0,0 +1,55 @@
+/** @odoo-module */
+
+import { browser } from "@web/core/browser/browser";
+import { registry } from "@web/core/registry";
+import { StatusBarField } from "@web/views/fields/statusbar/statusbar_field";
+import { useService } from "@web/core/utils/hooks";
+import { useState } from "@odoo/owl";
+
+const { onWillStart } = owl;
+
+
+export class StatusBarTimeField extends StatusBarField {
+    setup() {
+        super.setup();
+        this.user = useService("user");
+        onWillStart(async () => {
+            this.showTimeOptions = await this.user.hasGroup("crm.group_use_time_per_stage");
+        });
+        this.state = useState({ showTimePerStage: browser.localStorage.getItem('showTimePerStage') === 'true' });
```

Comment:
maybe something like showTimeState would help prevent unexpected collisions, if this is not included in `StatusBarField` in the end?